vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md

```diff
@@ -146,7 +146,7 @@ You can now deploy your Entando applications to GKE.
 ### Setup and Deploy
 1. Download and unpack the entando-helm-quickstart release you want to use from here:
 <https://github.com/entando-k8s/entando-helm-quickstart/releases>
-   - e.g. `curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.2.0.tar.gz | tar xvz`
+   - e.g. `curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.3.0.tar.gz | tar xvz`
```

Comment:
We'll conflict here. I restructured this to match the other command formats.

Reply:
Pulled in your changes to the gke doc.